templates/patterns/card/pattern-card.html.twig

```diff
@@ -40,6 +40,7 @@
     gutter: 0,
   } : {},
   'title': title,
+  'title_tag': title_tag,
```

Comment:
correct me if I'm wrong but I dont see the title tag tested in the kernel pattern test for the card

Reply:
done. Also added title_tag for search variant